src/Shared/BenchmarkRunner/DefaultCoreConfig.cs

```diff
@@ -30,12 +30,14 @@ public DefaultCoreConfig()
             AddJob(Job.Default
 #if NETCOREAPP2_1
                 .WithToolchain(CsProjCoreToolchain.From(NetCoreAppSettings.NetCoreApp21))
-#elif NETCOREAPP3_0
+#elif NETCOREAPP3_1
                 .WithToolchain(CsProjCoreToolchain.From(new NetCoreAppSettings("netcoreapp3.0", null, ".NET Core 3.0")))
 #elif NETCOREAPP3_1
                 .WithToolchain(CsProjCoreToolchain.From(new NetCoreAppSettings("netcoreapp3.1", null, ".NET Core 3.1")))
 #elif NETCOREAPP5_0
```

Comment:
Shouldn't this and the next `#elif` be `NET5_0` and `NET6_0`❔ Not sure anything defines `NETCORE6_0`.